### ISSUE TITLE
运用prefix_projection 方法训练test acc不变一直是62.1

### ISSUE THREAD

**Issue by TheMrYang** [THUDM/P-tuning-v2#13]
12/07/2021 19:30:19 - INFO - training.trainer_base - ***** Epoch 12: Best results *****                                                     12/07/2021 19:30:19 - INFO - training.trainer_base - best_epoch = 0
12/07/2021 19:30:19 - INFO - training.trainer_base - best_eval_accuracy = 0.6217125382262997
12/07/2021 19:30:19 - INFO - training.trainer_base - epoch = 12.0
OrderedDict([('best_epoch', 0), ('best_eval_accuracy', 0.6217125382262997), ('epoch', 13.0)])
{'loss': 0.7488, 'learning_rate': 0.006054054054054054, 'epoch': 13.51}

**Comment by Xiao9905** [comment]
Hi, prefix projection needs a careful hyper-parameter tuning, and in practice we find it does not work for every tasks.

**Comment by Xiao9905** [comment]
As we stated in the README.md, the hyperparameters provided in the repo is reimplemented with 3090 GPUs, rather than A100s (on which our paper results come from). 
`Since experiments reported in our paper are all conducted on NVIDIA DGX-A100 servers (which might be difficult to acquire), we reimplement P-tuning v2's results on BERT-large/RoBERTa-large with:`

**Comment by TheMrYang** [comment]
> As we stated in the README.md, the hyperparameters provided in the repo is reimplemented with 3090 GPUs, rather than A100s (on which our paper results come from). `Since experiments reported in our paper are all conducted on NVIDIA DGX-A100 servers (which might be difficult to acquire), we reimplement P-tuning v2's results on BERT-large/RoBERTa-large with:`

thanks,I think it will only affect the speed. Does it have anything to do with the result？

**Comment by Xiao9905** [comment]
As is in the README.md
`We notice that the best hyper-parameters can be sensitive to your server environment and package version.`
